tests/renderer/components/settings-spec.tsx

```diff
@@ -118,15 +118,15 @@ describe('Settings component', () => {
 
     // trigger mock 'keyup' event
     map.keyup({ code: 'Escape' });
-    expect(Object.keys(map)).toHaveLength(1);
+    expect(Object.keys(map)).toHaveLength(2); // {'keyup','contextmenu'}
     expect(store.isSettingsShowing).toBe(true);
 
     // Toggle the setting again as if it was closed
     instance.toggleHasPopoverOpen();
 
     // trigger mock 'keyup' event
     map.keyup({ code: 'Escape' });
-    expect(Object.keys(map)).toHaveLength(1);
+    expect(Object.keys(map)).toHaveLength(2); // {'keyup','contextmenu'}
     expect(store.isSettingsShowing).toBe(false);
 
     // check if event listener is removed upon unmount
```

Comment:
Let's update this comment to be "// check if event listeners are removed upon unmount" now that there's multiple.